File: src/main/java/com/alvcohen/travelhelper/TravelHelperApplication.java
package com.alvcohen.travelhelper;

import com.alvcohen.travelhelper.config.AppProperties;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;

@SpringBootApplication
@EnableConfigurationProperties(AppProperties.class)
public class TravelHelperApplication {
    public static void main(String[] args) {
        SpringApplication.run(TravelHelperApplication.class, args);
    }
}

File: src/main/kotlin/com/alvcohen/travelhelper/TravelHelperApplication.kt
package com.alvcohen.travelhelper

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
open class TravelHelperApplication {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            runApplication<TravelHelperApplication>(*args)
        }                                               
    }
}
                        